app/client/components/cancel/stages/executeCancellation.tsx

```diff
@@ -72,14 +72,15 @@ const getCancellationSummaryWithReturnButton = (
 const getCaseUpdatingCancellationSummary = (
   caseId: string,
   productType: ProductTypeWithCancellationFlow
-) => (withSubscription: WithSubscription | {}) => {
-  const subscription =
-    (withSubscription as WithSubscription).subscription || {};
+) => (productDetail: ProductDetail) => {
```

Comment:
looks like this has been broken a while, but it should be like this...
```suggestion
) => (productDetails: ProductDetail[]) => {
const productDetail = productDetails[0];
```
... because the API returns an array of one item.